.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "c:\\Work\\Hackathon-Login-Starter-Pack\\venv\\Scripts\\python.exe"
+}
```

Comment:
This would fail in a linux machine

Reply:
Wait .vscode/ shouldnt come it should be gitignored. Let me check.